### PR TITLE
BUG: Fix mapclassify #88

### PR DESCRIPTION
**Pull request by jeffcsauer**
This is a PR to fix https://github.com/pysal/mapclassify/issues/88. Changes have been made to [`_classify()` function](https://github.com/jeffcsauer/mapclassify/blob/7aad6fca0521cc29c64d43a1618dabd16a1ea965/mapclassify/classifiers.py#L632-L637) such that it now checks to see if max `y` and min `y` are equal (across multiple classifiers, not just `EqualInterval`). If max `y` and min `y` are equal there is now a `ValueError` with the following description: 

`Minimum and maximum of input data are equal, cannot create bins.`

There may be a better error message...

PR as part of the October 2020 PySAL Sprint.

@sjsrey

### CODE REVIEW

**Review by slumnitz — APPROVED** [review]
success! 👍

File: mapclassify/classifiers.py [hunk 1]
```diff
@@ -630,8 +630,11 @@ def _summary(self):
...
-        self._set_bins()
-        self.yb, self.counts = bin1d(self.y, self.bins)
+        if min(self.y) == max(self.y):
+            raise ValueError("Minimum and maximum of input data are equal, cannot create bins.")
+        else:
+            self._set_bins()
+            self.yb, self.counts = bin1d(self.y, self.bins)
 
     def _update(self, data, *args, **kwargs):
         """
```
Comment (sjsrey) [inline mapclassify/classifiers.py:637]: This looks good and handles this edge case.

I might add some unit tests.

**Review by sjsrey — CHANGES_REQUESTED** [review]

**Comment by sjsrey** [comment]
Should be passing now that 3.6 is excluded on windows with #94
